pkg/server/api/audit/audit.go

```diff
@@ -0,0 +1,72 @@
+package audit
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/spiffe/spire-api-sdk/proto/spire/api/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type Log interface {
+	AddFields(logrus.Fields)
+	Emit(logrus.Fields)
+	EmitBatch(*types.Status, logrus.Fields)
+	EmitError(error)
+}
+
+type log struct {
+	fields logrus.Fields
+	log    logrus.FieldLogger
+}
+
+func New(l logrus.FieldLogger) Log {
+	return &log{
+		log: l.WithFields(logrus.Fields{
+			"type": "audit",
+			// It is success by default, erros must change it
+			"status": "success",
+		}),
+		fields: logrus.Fields{},
+	}
+}
+
+func (l *log) AddFields(fields logrus.Fields) {
+	for key, value := range fields {
+		l.fields[key] = value
+	}
+}
+
+func (l *log) Emit(fields logrus.Fields) {
+	l.log.WithFields(l.fields).WithFields(fields).Info("Audit log")
```

Comment:
Given how frequently this will be emitted and the fact that we already have the `type` field set to `audit`, is there any benefit to logging this "Audit log" message? It seems like it is redundant data.

Since this audit logger is specific to the API layer, maybe something like "API accessed" would be more appropriate?